bridger-agent/src/components/TransactionTable/FilterableDropdown.tsx

```diff
@@ -0,0 +1,235 @@
+import { useRef, useState, useEffect } from "react";
```

Comment:
Note from reviewer: This is part of our prod design components that I copied over